src/hooks/useSetupPlayer.ts

```
@@ -75,13 +81,21 @@ export default ({ intentData }: NoxComponent.AppProps) => {
         case IntentData.PlayAll:
           // this hook cannot use usePlayback bc of rerendering..??
           break;
-        default:
+        case undefined:
           await TrackPlayer.pause();
```

Biome

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)